collection-scripts/gather_network_logs

```diff
@@ -50,6 +71,18 @@ function gather_ovn_kubernetes_data {
       ((i++))
     done
   done
+}
+
+function gather_ovn_kubernetes_data {
+  sample_node=$(oc get no -o jsonpath='{.items[0].metadata.name}')
+  sample_node_zone=$(oc get node "${sample_node}" -o jsonpath='{.metadata.annotations.k8s\.ovn\.org/zone-name}')
+  if [ "${sample_node}" = "${sample_node_zone}" ]; then
```

Comment:
nice! I think this is good if we never go multi nodes per zone; which is the case.